components/game/game.js

```diff
@@ -1,12 +1,18 @@
+import React, { useRef, useState, useEffect } from 'react';
+import { Slackey } from 'next/font/google';
 import gameStyles from './game.module.scss';
 import styles from "@/app/page.module.css";
-import React, { useState, useEffect } from 'react';
-import Hand from "@/components/gameComponents/hand";
+import Hand from "@/components/hand/hand";
 import Cards from "@/components/cards/cards";
 import { dictionaryCombinations, highestValue } from '@/components/utilities/combination';
 import { mapCard, icons } from '../utilities/card';
 import { aiMoves, aiPossibleCombinations, determineHardestMove } from '../utilities/ai';
 
+const slackey = Slackey({
+  weight: '400',
+  subsets: ['latin'],
+})
```

Comment:
what does this do? also add a semicolon at the end

```suggestion
const slackey = Slackey({
  weight: '400',
  subsets: ['latin'],
});
```

Reply:
Was going to add a font family. Removing now.